src/rpc/rawtransaction.cpp

```diff
@@ -35,6 +35,11 @@
 
 using namespace std;
 
+#ifdef ENABLE_WALLET
+std::string HelpRequiringPassphrase(CWallet *);
```

Comment:
I suppose we could. But then it's more likely to get used in new code (which I think we want to discourage?)